e2e/protractor.excludes.json

```diff
@@ -3,6 +3,7 @@
   "C362240": "Include once ADF starts using ACS 7 , https://issues.alfresco.com/jira/browse/ADF-5182",
   "C362241": "Include once ADF starts using ACS 7, https://issues.alfresco.com/jira/browse/ADF-5182",
   "C362242": "Include once ADF starts using ACS 7, https://issues.alfresco.com/jira/browse/ADF-5182",
-  "C362265": "Include once ADF starts using ACS 7, https://issues.alfresco.com/jira/browse/ADF-5182"
+  "C362265": "Include once ADF starts using ACS 7, https://issues.alfresco.com/jira/browse/ADF-5182",
+  "C310358": "https://issues.alfresco.com/jira/browse/AAE-3262"
```

Comment:
why are we excluding this test?

Reply:
This PR will fix an issue: if destinationFolder is set to -root-, when the user opens the attach file widget dialog will see in the destination picker will open 'Company Home' folder, but this test expects to have users folder[this is how it was before]. So the plan is: we are excluding this test, merging this PR, change the app to use -my- instead of -root- as destination Folder in AMA, then re enable the test again.

Reply:
We modified the app and re-enabled the test